src/main/java/Joe.java

```diff
@@ -0,0 +1,84 @@
+import java.util.Scanner;
+
+public class Joe {
+    protected static final String EXIT_COMMAND = "bye";
+    protected static final String LIST_COMMAND = "list";
+    protected static final String MARK_COMMAND = "mark";
+    protected static final String UNMARK_COMMAND = "unmark";
+    protected static final String NEW_TODO_COMMAND = "todo";
+    protected static final String NEW_DEADLINE_COMMAND = "deadline";
+    protected static final String NEW_EVENT_COMMAND = "event";
+
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        Printer.printGreeting();
+
+        TaskManager taskManager = new TaskManager();
+        boolean hasExitInput = false;
+        while (!hasExitInput) {
```

Comment:
Can consider moving while loop to a separate function so main function is not as cluttered